djautotask/sync_rest.py

```diff
@@ -563,3 +563,94 @@ def _assign_field_data(self, instance, json_data):
     def get_page(self, next_url=None, *args, **kwargs):
         kwargs['conditions'] = self.api_conditions
         return self.client.get_tasks(next_url, *args, **kwargs)
+
+
+class ProjectSynchronizer(SyncRestRecordUDFMixin, Synchronizer,
+                          ParentSynchronizer):
+    client_class = api.ProjectsAPIClient
+    model_class = models.ProjectTracker
+    udf_class = models.ProjectUDF
+    # override
```

Comment:
This is me being probably too picky. But I think this comment is unnecessary

Reply:
All is applied again. ~